packages/frontend/app/pods/components/pod/chat/chat-entry/component.ts

```diff
@@ -59,6 +60,7 @@ export default class ChatEntry extends Component<IArgs> {
     if (!this.text) return;
 
     this.isDisabled = true;
+    this.text = unicode(this.text);
```

Comment:
to make the unicode live update as you type, you could invoke the unicode function on line 84